doc/itof/nxp_build_instructions.md

```diff
@@ -68,6 +69,18 @@ sudo cmake --build . --target install
 popd
 ```
 
+* Depth Compute:
+
+The Depth Compute library is required to build the SDK and run any application based on the SDK.
+Based on the following scenarios you will be able to do the following:
+
+1. You have the **arm development package for nxp** (libtofi_compute.so, libtofi_config.so) for the Depth Compute library. With it you will be able to build the SDK. You just need to put the files mentioned above in a directory called 'libs' which should be placed at the same level as the repository directory.
+
+    If you prefer to keep the files somewhere else, you can tell cmake where to find them using the cmake variable: LIBTOFI_LIBDIR_PATH
+
+2. You have the **distribution package** (tofi_compute.so, tofi_config.so) for the Depth Compute library. With it you won't be able to build the SDK unless you turn on the  USE_DEPTH_COMPUTE_OPENSOURCE cmake option. This will create an empty Depth Compute library allowing the build of SDK to complete. To use the real Depth Compute, just replace the stubs generated during build (tofi_compute.so, tofi_config.so) with the ones from the distribution package.
+
+3. You have no Depth Compute packages. In this case you can still build the SDK by enabling the USE_DEPTH_COMPUTE_OPENSOURCE option but you won't have access to the functionality that Depth Compute provides.
```

Comment:
I wouldn't keep item 2.
Also I would rephrase item 1 to not mention about the 'arm development package for nxp'. But I would mention that the  library should included in the SD card image. Please check with @NKarnat about the exact path where the depth compute .so files will be in the next release.

Reply:
Solved